capstone/src/main/java/com/google/sps/servlets/PrototypeStudents.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+/** Hard-coded student data for prototype */
+public final class PrototypeStudents {
+  public static final String MEGAN_EMAIL = "meghashi@google.com";
```

Comment:
nit: don't think this is quite right :)

Reply:
LOL how did I not catch that, thanks Chris!